source/TestCommon/source/FunctionApp.TestCommon/OpenIdJwt/OpenIdJwtManager.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Security.Cryptography;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.Configuration;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.TestCertificate;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Energinet.DataHub.Core.FunctionApp.TestCommon.OpenIdJwt;
+
+/// <summary>
+/// Used to help test DH3 applications that requires OpenId and JWT for HTTP authentication and authorization.
+/// The OpenIdJwtManager supports:
```

Comment:
Reduce chance of any renaming causing comments to be "incorrect".

```suggestion
/// The manager supports:
```